.github/workflows/test.yml

```
@@ -42,4 +42,4 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-targets -- -D warnings
+          args: --all-targets
```

Do you still need to remove this if we have `#![allow(clippy::from_iter_instead_of_collect)]` in the appropriate places? It's probably best to deny warnings - it prevents things like unused imports, etc.

If adding it back in still causes a failure, maybe try:

```suggestion
          args: --all-targets -- -D warnings -A clippy::from_iter_instead_of_collect
```

---

The `from_iter_instead_of_collect` lint is an unstable flag and only available on nightly, the problem is that it is complex/impossible to only allow it on nightly.